modules/mapping-utils/src/createConnectionResolvers.js

```diff
@@ -13,11 +13,6 @@ let createConnectionResolvers: TcreateConnectionResolvers = ({ type }) => ({
       // TODO: stitch extended mapping
       return type.mapping;
     },
-    extended: (obj, { fields }) => {
```

Comment:
is this on purpose?

Reply:
This is part of the normalization commit